python/lsst/ts/gis/enums.py

```diff
@@ -0,0 +1,1175 @@
+__all__ = [
+    "gisCpuInputs",
+    "gisCpuOutputs",
+    "gisCpuReserve",
+    "afeDecentralizedIOInputs",
+    "afeDecentralizedIOOutputs",
+    "afeDecentralizedIOFree",
+    "laserDecentralizedIOInput",
+    "laserDecentralizedIOOutputs",
+    "laserDecentralizedIOFree",
+    "m2cDecentralizedIOInput",
+    "m2cDecentralizedIOOutput",
+    "m2cDecentralizedIOFree",
+    "pfDecentralizedIoInputs",
+    "pfDecentralizedIoOutput",
+    "pfDecentralizedIoFree",
+    "auxCpuInputs",
+    "auxCpuOutputs",
+    "domeCpuInputs",
+    "domeCpuOutputs",
+    "m1m3CpuInputs",
+    "m1m3CpuOutputs",
+    "tmaCpuInputs",
+    "tmaCpuOutputs",
+    "causes",
+    "causes2",
+    "causesOverride",
+    "causes2Override",
+    "effects",
+    "effects2",
+    "subsystem_order",
+]
+
+from dataclasses import dataclass
+
+subsystem_order = (
+    "gisCpuInputs",
+    "gisCpuOutputs",
+    "gisCpuReserve",
+    "afeDecentralizedIOInputs",
+    "afeDecentralizedIOOutputs",
+    "afeDecentralizedIOFree",
+    "laserDecentralizedIOInput",
+    "laserDecentralizedIOOutputs",
+    "laserDecentralizedIOFree",
+    "m2cDecentralizedIOInput",
+    "m2cDecentralizedIOOutput",
+    "m2cDecentralizedIOFree",
+    "pfDecentralizedIoInputs",
+    "pfDecentralizedIoOutput",
+    "pfDecentralizedIoFree",
+    "auxCpuInputs",
+    "auxCpuOutputs",
+    "domeCpuInputs",
+    "domeCpuOutputs",
+    "m1m3CpuInputs",
+    "m1m3CpuOutputs",
+    "tmaCpuInputs",
+    "tmaCpuOutputs",
+    "causes",
+    "causes2",
+    "causesOverride",
+    "causes2Override",
+    "effects",
+    "effects2",
+)
+
+
+@dataclass
+class gisCpuInputs:
+    """Main GIS CPU inputs.
+
+    Parameters
+    ----------
+    sdiCPUetw1A:
+        Wireless emergency pushbutton 1 channel 1.
+    sdiCPUetw1B:
+        Wireless emergency pushbutton 1 channel 2.
+    sdiCPUetw2A:
+        Wireless emergency pushbutton 2 channel 1.
+    sdiCPUetw2B:
+        Wireless emergency pushbutton 2 channel 2.
+    sdiCPUfree1:
+        Reserve.
```

Comment:
I think we talked about this before. With `Reserve` you mean `Reserved for future use`, right? Please change here and everywhere else in this enums.py file.